src/Components/Pages/ManageProfilepage.js

```diff
@@ -0,0 +1,56 @@
+import { Button, Card, Container, Form } from 'react-bootstrap';
+import { useNavigate } from "react-router-dom";
+
+
+function ManageProfilePage() {
+
+    let navigate = useNavigate();
+
+    return (
+        <Container fluid style={{display: 'flex', justifyContent: 'center' }}>
+            
+            
+            <h1 onClick={() => {navigate('/')}} style={{textAlign: "center", marginTop: '3%', fontWeight:"bold"}}>BOREAL.ca</h1>
+            
+            <Card style={{ width: '18rem', position: 'absolute', top: '15%', outline: "solid" }}>
+            <h3 style={{ textAlign: "center" }}>Edit Account Information</h3>
```

Comment:
Consider adding a top margin of 5% for the `Edit Account Information` text.
This can be done by adding `marginTop: 5%`.

Alternatively, you could include the `h3` element in the card body and add a bottom margin of 10%.
This can be done by writing `marginBottom: 10%`.